src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedMathUtils.java

```diff
@@ -33,7 +35,18 @@ public static double naturalLogSumExp(final double ... values) {
         }
         return max + Math.log(sum);
     }
-    
+
+    public static double naturalLogSumExp(final ArrayList<Double> values) {
```

Comment:
I rewrote with collection. The usage of "naturalLog" is for semantic compatibility with David's original method `naturalLogSumExp(final double ... values)`. I will rename both.